eZ/Bundle/PlatformInstallerBundle/src/Resources/config_templates/clean/ezpublish.yml

```diff
@@ -21,6 +21,7 @@ ezpublish:
         # Available siteaccesses
         list:
             - site
+            - mobile
```

Comment:
Ah, that. Yes, it is the real config for the `clean` install.

Maybe we should copy the clean install under a new name to clarify things ? Or try to take advantage of it to "remind" what siteaccesses are for ? But with responsive...

Fact is that I need an extra siteaccess in here :-)